packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalItemGroup as ItemGroup,
+	Button,
+	FlexItem,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+import { Icon, plus, shadow as shadowIcon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import Subtitle from './subtitle';
+import { NavigationButtonAsItem } from './navigation-button';
+import ScreenHeader from './header';
+
+const { useGlobalSetting } = unlock( blockEditorPrivateApis );
+
+export const defaultShadow = '6px 6px #000';
+
+export default function ShadowsPanel() {
+	const [ defaultShadows ] = useGlobalSetting( 'shadow.presets.default' );
+	const [ themeShadows ] = useGlobalSetting( 'shadow.presets.theme' );
+	const [ customShadows, setCustomShadows ] = useGlobalSetting(
+		'shadow.presets.custom'
+	);
+
+	const onCreateShadow = ( shadow ) => {
+		setCustomShadows( [ ...( customShadows || [] ), shadow ] );
+	};
+
+	return (
+		<>
+			<ScreenHeader
+				title={ __( 'Shadows' ) }
+				description={ __(
+					'Shadows and the application of those shadows on site elements.'
+				) }
+			/>
+			<div className="edit-site-global-styles-screen">
+				<VStack
+					className="edit-site-global-styles-shadows-panel"
+					spacing={ 7 }
+				>
+					<ShadowList
+						label={ __( 'Default' ) }
+						shadows={ defaultShadows || [] }
+						category={ 'default' }
+					/>
+					<ShadowList
+						label={ __( 'Theme' ) }
+						shadows={ themeShadows || [] }
+						category={ 'theme' }
+						placeholder={ __( 'Theme shadows are empty!' ) }
+					/>
+					<ShadowList
+						label={ __( 'Custom' ) }
+						shadows={ customShadows || [] }
+						category={ 'custom' }
+						placeholder={ __( 'Custom shadows are empty!' ) }
+						onCreate={ onCreateShadow }
+					/>
+				</VStack>
+			</div>
+		</>
+	);
+}
+
+function ShadowList( { label, placeholder, shadows, category, onCreate } ) {
+	const handleAddShadow = () => {
+		onCreate( {
+			name: `Shadow ${ shadows.length + 1 }`,
```

Comment:
Probably "Shadow" should be translatable.

Reply:
It is a name given by user such as `My custom shadow`, and when creating a first shadow giving it a default name `Shadow n`. So, it is not a candidate for translations.